stage1_fly/aci/aci-manifest.in

```diff
@@ -0,0 +1,33 @@
+{
+    "acKind": "ImageManifest",
+    "acVersion": "0.7.3",
+    "name": "coreos.com/rkt/stage1-fly",
+    "labels": [
+        {
+            "name": "version",
+            "value": "@RKT_STAGE1_VERSION@"
+        },
+        {
+            "name": "arch",
+            "value": "amd64"
```

Comment:
Good point. stage1 from src and host could also support other architectures so we need a generic solution.